core/src/impl/Kokkos_ExecSpaceManager.hpp

```diff
@@ -74,7 +74,7 @@ struct ExecSpaceFooDerived : ExecSpaceFooBase {
     ExecutionSpace().fence(std::move(label));
   }
   void print_configuration(std::ostream& msg, bool detail) final {
-    ExecutionSpace::print_configuration(msg, detail);
+    ExecutionSpace().print_configuration(msg, detail);
```

Comment:
Why did you change that guy rather than update the backends?

Reply:
`print_configuration` is not supposed to be `static`.

Reply:
There are a couple of pull requests explicitly replacing `static` use of `print_configuration`: https://github.com/kokkos/kokkos/pull/4806, https://github.com/trilinos/Trilinos/pull/10265
and https://github.com/kokkos/kokkos/wiki/ExecutionSpaceConcept lists it as non-`static`.
Both `OpenMPTarget` and `SYCL` implement non-`static` versions. For `SYCL` the information printed depend on the device being used and I was glad that I could do that.

If we want to change that, we should discuss it elsewhere.